datacache/database_table.py

```diff
@@ -76,7 +76,7 @@ def from_fasta_dict(cls, name, fasta_dict, key_column, value_column):
 
         def make_rows():
             return [
-                (idx, str(record.seq))
+                {key_column: idx, value_column: str(record.seq)}
```

Comment:
This needed to be changed to accommodate this switch, which takes different input: https://github.com/hammerlab/datacache/pull/25/files#diff-9ec530bc1370e34610cdd63c31e2f252R187